cloud.build.yaml

```diff
@@ -1,5 +1,13 @@
 steps:
   - name: "gcr.io/cloud-builders/npm"
+    id: install
     args: ["install"]
   - name: "gcr.io/cloud-builders/npm"
+    waitFor: ["install"]
     args: ["run", "test"]
+  - name: "gcr.io/cloud-builders/npm"
+    waitFor: ["install"]
+    args: ["run", "typecheck"]
+  - name: "gcr.io/cloud-builders/npm"
+    waitFor: ["install"]
+    args: ["run", "format:check"]
```

Comment:
Thinking we should try to squeeze in as much QA stuff as possible here (within reason). Maybe we should call it something different than "build' also - `cloud.qa.yaml`?